types/jsx-pdf/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true,
+        "jsx": "react",
+        "jsxFactory": "JsxPdf.createElement"
```

Comment:
@orta @elibarzilay we use this option at all ?

Reply:
Yeah, I don't think tsc could compile the test .d.ts otherwise because the JS emit would be: `React.createElement` when there isn't a `React` symbol available in the file.